includes/admin/class-wc-stripe-admin-notices.php

```
@@ -472,7 +472,12 @@ public function subscription_check_detachment() {
 		}
 
 		if ( ! $subscription->has_status( [ 'active' ] ) ) {
-			// Only show the notice for active subscriptions.
+			return;
+		}
+
+		$dismissed_subscriptions = get_option( 'wc_stripe_show_subscription_detached_notice', [] );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $dismissed_subscriptions. Keep variable name length under 20. (undefined)

(LongVariable)